lib/steps/load_state.go

```diff
@@ -5,18 +5,41 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"os"
 )
 
-func loadState(command string) RunState {
+func hasSavedState(command string) bool {
 	filename := getRunResultFilename(command)
-	content, err := ioutil.ReadFile(filename)
+	_, err := os.Stat(filename)
 	if err != nil {
-		log.Fatal(err)
+		if os.IsNotExist(err) {
```

Comment:
This is okay for now. Let's see what `golint` thinks about it. Btw is your editor running Golint when you save the file? This is super useful in my experience.